membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -153,9 +153,11 @@ object AccountDetails {
           "shouldDisplayEmail" -> selfServiceCancellation.shouldDisplayEmail,
           "phoneRegionsToDisplay" -> selfServiceCancellation.phoneRegionsToDisplay,
         ),
+        "billingCountry" -> billingCountry.iterator.mkString,
       ) ++
         regNumber.fold(Json.obj())({ reg => Json.obj("regNumber" -> reg) }) ++
-        Json.obj(
+        billingCountry.fold(Json.obj())({ bc => Json.obj("billingCountry" -> bc.name)}) ++
```

Comment:
Are we adding billiingCountry twice, once as a string and once as a json object?